CHANGELOG.md

```diff
@@ -1,78 +1,144 @@
 # Changelog
 
-## 2017-01-13 - Release 2.1.0
+All notable changes to this project will be documented in this file.
+Each new release typically also includes the latest modulesync defaults.
+These should not affect the functionality of the module.
 
-This is the last release with Puppet 3 support!
```

Comment:
It's a shame to have lost this line.  @bastelfreak Do you mind if I create a HISTORY.md and regenerate for 3.0.1?

Reply:
BTW, I've come up with a good reason for a 3.0.1 release :) https://github.com/voxpupuli/puppet-mrepo/pull/78